src/Dictionaries/HashedArrayDictionary.cpp

```diff
@@ -822,10 +822,12 @@ void HashedArrayDictionary<dictionary_key_type, sharded>::loadData()
         if (parallel_loader)
             parallel_loader->finish();
 
-        LOG_DEBUG(getLogger("HashedArrayDictionary"),
-            "Finished {}reading {} blocks with {} rows from pipeline in {:.2f} sec and inserted into hashtable in {:.2f} sec",
+        LOG_DEBUG(log,
+            "Finished {}reading {} blocks with {} rows to dictionary {} from pipeline in {:.2f} sec and inserted into hashtable in {:.2f} sec",
```

Comment:
```suggestion
            "Finished {}reading {} blocks with {} rows to dictionary `{}` from pipeline in {:.2f} sec and inserted into hashtable in {:.2f} sec",
```

Reply:
It returns name for logs from https://github.com/ClickHouse/ClickHouse/blob/4a7ad15a9a7cf9094ff44a010ae08386036ada9a/src/Interpreters/StorageID.cpp#L60-L66, so ticks are excessive here